### ISSUE TITLE
NPE after selecting an image from gallery/library (4.0.2)

### ISSUE THREAD

**Issue by skolarianer** [smooch/smooch-android#39]
When I try to share an image from gallery (library) I get a NullPointerException. The gallery opens, but when I click on an image the app crashes. Using "last photo" works like a charm.

```
java.lang.NullPointerException: Attempt to invoke virtual method 'int android.graphics.Bitmap.getWidth()' on a null object reference
  at io.smooch.ui.fragment.ConversationFragment.scaleBitmap(Unknown Source)
  at io.smooch.ui.fragment.ConversationFragment.smartDecode(Unknown Source)
  at io.smooch.ui.fragment.ConversationFragment.access$1100(Unknown Source)
  at io.smooch.ui.fragment.ConversationFragment$17.run(Unknown Source)
  at android.os.AsyncTask$SerialExecutor$1.run(AsyncTask.java:231)
  at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1112)
  at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:587)
  at java.lang.Thread.run(Thread.java:818)
```

**Comment by jugarrit** [comment]
I'll take a look at this ASAP!

**Comment by jugarrit** [comment]
Oversight on my part. This will be fixed in the next release!

**Comment by jugarrit** [comment]
This should now be fixed in 4.0.3
